app/scripts/controllers/mobileClients.js

```diff
@@ -32,6 +33,39 @@ angular.module('openshiftConsole')
           title: $routeParams.mobileclient
         }
       ];
+      ctrl.editDmzUrl = false;
+
+      ctrl.setDmzUrl = function(dmzUrl) {
+        var mobileClient = angular.copy(ctrl.mobileClient);
+        if (dmzUrl && dmzUrl != "") {
```

Comment:
not sure if they prefer the lodash version of this i.e. `_.isEmpty()`

Reply:
Was actually supposed to remove that. Good catch :+1: